packages/palette/src/elements/Image/LazyImage.tsx

```diff
@@ -83,8 +85,8 @@ export const LazyImage: React.FC<LazyImageProps> = ({
         src={src}
         title={title}
         alt={alt}
-        ariaLabel={ariaLabel}
-        borderRadius={borderRadius}
+        aria-label={ariaLabel}
+        borderradius={borderRadius}
```

Comment:
Hmmm... I don't think this is how to solve the problem. 

Ultimately, the error is that the `Image` element that styled-components is rendering is getting `ariaLabel` passed to it and react is complaining about that. Your change might stop that error from happening, but it'll break the image component because the props are different. And the `aria-label` is invalid JSX I believe. 

I've had it on my todo list to fix this problem and I can do so in a separate PR.

Reply:
Cool, I'll rebase that commit out and defer to #355 for this fix. 👍